specification/monitor/data-plane/preview/2018-09-01-preview/MetricsPost_API.json

```diff
@@ -21,8 +21,11 @@
   "paths": {
     "/subscriptions/{subscriptionId}/resourcegroups/{resourceGroupName}/providers/{resourceProvider}/{resourceTypeName}/{resourceName}/metrics": {
       "post": {
-        "summary": "Ingestion API used by Azure custom metrics\r\n            Responsible of below operation\r\n            1. Authorize the request\r\n            2. Deserialize payload request body\r\n            3. Perform validation of the payload\r\n            4. Commits the payload for metrics ingestion",
-        "operationId": "azureMonitorCustomMetricsIngestionApi",
+        "tags": [
+          "Metrics"
+        ],
+        "operationId": "Metrics_POST",
```

Comment:
The underscore in the name is going to create an API group named `Metrics` with one API named `POST`.  I don't think this is the best naming.  What about `Metrics_Create`?  If this API can be used to update existing data then it should be `Metrics_CreateOrUpdate`.

Reply:
It can be called create. We already have a Metrics_List and this should go along side of that. Sure i can make it "Create" to make it consistent.